### ISSUE TITLE
base images on cudagl (for openGL support)

### ISSUE THREAD

**Issue by AwokeKnowing** [floydhub/dockerfiles#43]
nvidia docker version 2 eg (docker run --runtime=nvidia ...)  had not worked with opengl until a few days ago.   So for example, we cannot run the open ai gym environments which rely on opengl.

nvidia just released a new cuda 9 image with open gl.  this is great news especially for people using gym environment and ROS environments.

Here you can read a bit about the issue progress
https://github.com/NVIDIA/nvidia-docker/issues/534

Here's the nvidia link
https://hub.docker.com/r/nvidia/cudagl/

I'm so excited about this! I'm resisting the urge to clone and rebuild all of your images just for the pytorch image!   And by the way, I see you've been updating gym recently so this should go right along with it.

**Comment by houqp** [comment]
Yes, we are about to rebuild everything with the new cuda 9 image :)

**Comment by houqp** [comment]
So I looked into cudagl. Turns out it does not come with cudnn, so we can't use it for our base right now. We need to manually install cudnn into cudagl or install opengl into cuda base. I am trying to figure out if there is an easy way to do this without copy pasting.

**Comment by karray** [comment]
@houqp any news about cuda + cudnn + opengl?
